index.html

```diff
@@ -28,16 +28,148 @@
     <section class="headline-flex">
         <div class="headline-heading-placeholder">
             <h1 class="headline-heading-name">Gabriel Fonseca</h1>
-            <h2 class="headline-heading-occupation">Software Developer</h2>
+            <h3 class="headline-heading-occupation">Software Developer</h3>
         </div>
         <div class="headline-description-placeholder">
             <p class="headline-description">I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
         </div>
     </section>
+
+    <div class="section-separation"></div>
+
+    <div class="about-me-flex">
+      <div class="about-me-languages">
+        <div class="about-me-title">Language
+          <ul class="about-me-list">
+            <li class="about-me-list-item">Javascript</li>
+            <li class="about-me-list-item">Ruby</li>
+            <li class="about-me-list-item">Html</li>
+            <li class="about-me-list-item">Css</li>
+         </ul>
+      </div>
+      </div>
+      <div class="about-me-frameworks">
+        <div class="about-me-title">Frameworks
+          <ul class="about-me-list">
+            <li class="about-me-list-item">Bootstrap</li>
+            <li class="about-me-list-item">Ruby on Rails</li>
+            <li class="about-me-list-item">RSPec</li>
+            <li class="about-me-list-item">CapyBara</li>
+            <li class="about-me-list-item">Selenium</li>
+          </ul>
+        </div>
+      </div>
+      <div class="about-me-skills">
+        <div class="about-me-title">Skills
+          <ul class="about-me-list">
+          <li class="about-me-list-item">Codekit</li>
+          <li class="about-me-list-item">Github</li>
+          <li class="about-me-list-item">Codepen</li>
+          <li class="about-me-list-item">Gitlab</li>
+          <li class="about-me-list-item">Terminal</li>
+          </ul>
+        </div>
+      </div>
+    </div>
```

Comment:
- [ ] Your portfolio is looking great! Let's make it even better by using `ul` and `li` tags also for the technology categories, e.g:

```
<ul>
    <li>Languages
        <ul>
            <li>JS</li>
            <li>HTML</li>
            <li>CSS</li>
        </ul>
    </li>    
    <li>Frameworks
        <ul>
            <li>Bootsrap</li>
            <li>RoR</li>
            <li>RSpec</li>
        </ul>
    </li>

     . . .   

</ul>
```